src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
```

Comment:
The `main` method signature should not be followed by an empty line. Please remove the empty line to make the code more concise.